app/controllers/anon_routes_controller.rb

```diff
@@ -40,8 +42,8 @@ def create
     new_route.waypoints << Waypoint.create(latitude: wp2.lat, longitude: wp2.lng)
     new_route.waypoints << Waypoint.create(latitude: wp3.lat, longitude: wp3.lng)
 
-    url = "https://www.google.com/maps?saddr=#{start.latitude},+#{start.longitude}&daddr=#{wp1.latitude},+#{wp1.longitude}+to:#{wp2.latitude},+#{wp2.longitude}+to:#{wp3.latitude},+#{wp3.longitude}+to:#{start.latitude},+#{start.longitude}&dirflg=w"
-    puts url
+    new_route.map_url = "https://www.google.com/maps?saddr=#{start.latitude},+#{start.longitude}&daddr=#{wp1.latitude},+#{wp1.longitude}+to:#{wp2.latitude},+#{wp2.longitude}+to:#{wp3.latitude},+#{wp3.longitude}+to:#{start.latitude},+#{start.longitude}&dirflg=w"
+    new_route.save
```

Comment:
@ifricker @ajpage @michaelzwang @ficxin 
This controller action is fairly large, but the goal is to create the random route. I would like to see this moved to a `CreateRoute` service (name can be what you want).

Here's a little more on Services (https://www.netguru.co/blog/service-objects-in-rails-will-help)